roslibrust/src/ros1/node/actor.rs

```diff
@@ -164,6 +172,33 @@ impl NodeServerHandle {
         }
     }
 
+    pub async fn register_service_client<T: RosServiceType>(
+        &self,
+        service_name: &Name,
+    ) -> RosLibRustResult<mpsc::UnboundedSender<CallServiceRequest>> {
+        let (sender, receiver) = oneshot::channel();
+        match self
+            .node_server_sender
+            .send(NodeMsg::RegisterServiceClient {
+                reply: sender,
+                service: service_name.to_owned(),
+                service_type: T::ROS_SERVICE_NAME.to_owned(),
+                srv_definition: String::from_iter(
+                    [T::Request::DEFINITION, "\n", T::Response::DEFINITION].into_iter(),
+                ),
```

Comment:
The integration testing with roscpp and rospy will reveal whether this method of service definition generation is fine. I believe it should be.